linux/nvme.h

```diff
@@ -713,6 +713,29 @@ struct nvme_effects_log {
 	__u8   resv[2048];
 };
 
+struct nvme_lba_range_desc {
+	__le64	rslba;
+	__le32	rnlb;
+	__u8	rsvd12[4];
+};
+
+struct nvme_lba_status_ns_element {
+	__le32	neid;
+	__le32	nlrd;
+	__u8	ratype;
+	__u8	rsvd9[7];
+	struct nvme_lba_range_desc desc[];
+};
+
+struct nvme_lba_status_log_page {
+	__le32	lslplen;
+	__le32	nlslne;
+	__le32	estulb;
+	__u8	rsvd12[2];
+	__le16	lsgc;
+	struct nvme_lba_status_ns_element element;
```

Comment:
This is not a single element. It is a variable length array of size nlslne.

Reply:
This has been addressed and rebased it!